activesupport/lib/active_support/core_ext/time/zones.rb

```diff
@@ -82,7 +82,7 @@ def use_zone(time_zone)
     def find_zone!(time_zone)
       return time_zone unless time_zone
 
-      ActiveSupport::TimeZone[time_zone] || raise(ArgumentError, "Invalid Timezone: #{time_zone}")
+      ActiveSupport::TimeZone[time_zone] || raise(ArgumentError, "Invalid Timezone: #{time_zone}! Find the valid identifiers with: ActiveSupport::TimeZone::MAPPING")
```

Comment:
@ghiculescu: Wrapping such commands in ``` ` ``` often makes them harder to copy-for-pasting. With a `: ` before, and _no_ suffix, the separation is clearer, IMHO.

Or is there a detection mechanism for that command suggestion, so that it can be run without copy-pasting?

Reply:
👍 yeah I like this.